test/app.test.js

```diff
@@ -72,6 +72,19 @@ describe("app", function() {
 
         });
 
+        describe('Facility_Code_Confirm', function () {
+            it('should got to MSISDN entry', function () {
+                    return tester
+                        .setup.user.state('Facility_Code_Confirm')
+                        .input('1')
+                        .check.interaction({
+                            state: 'MSISDN_Entry',
+                            reply: /Please enter the South African cell phone number/
+                        })
+                        .run();
+            });
+        });
+
```

Comment:
Oops hit enter too early.

In the test we're priming the state, we tell it what state it is in (`Facility_Code_Confirm`) and then provide it some inputs (`1` in this case) and then verify that it results in the state transition to the next named state (`MSISDN_Entry`) and that the content returned from that next state matches the regex provided.